lib/ui/view/channel/edit_channel/edit_channel_view_model.dart

```diff
@@ -39,8 +41,11 @@ class EditChannelViewModel extends FormViewModel {
 
       return;
     }
-    const channel_id = '613f70bd6173056af01b4aba';
-    const endpoint = '$ChannelInfoEndpoint$channel_id/';
+    String _channelId = id;
+    String orgId = storage.getString(StorageKeys.currentOrgId).toString();
+    String endpoint = '/v1/'+orgId+'/channels/'+_channelId;
+    print(endpoint);
```

Comment:
Please use Logger and not print.
It should be `log.i()`

Reply:
resolved